SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityCleaner.java

```diff
@@ -362,9 +386,19 @@ private void drainTaskCleanupQueue() {
       return;
     }
 
+    Map<String, Integer> incrementalBounceRemainingInstanceMap = new HashMap<>();
```

Comment:
can you clarify what this map does? it tracks how many tasks still need to bounce per request?